src/transport/raw/BLE.h

```diff
@@ -48,8 +48,17 @@ class BleListenParameters
 
     Ble::BleLayer * GetBleLayer() const { return mLayer; }
 
+    bool HasOverrideExistingTransport() const { return mOverrideExistingTransport; }
+    BleListenParameters & SetOverrideExistingTransport(bool overrideExistingTransport)
+    {
+        mOverrideExistingTransport = overrideExistingTransport;
+
+        return *this;
+    }
+
 private:
     Ble::BleLayer * mLayer;
+    bool mOverrideExistingTransport = false;
```

Comment:
This changes previous behaviour. Is the previous behaviour wrong across the board? I think BLE used to work.

Reply:
When there is more than one TransportManager, BLE only works for one of them. This change is meant to address that. Added comments in https://github.com/project-chip/connectedhomeip/pull/15525